articles/blocks-and-scopes.html

```diff
@@ -443,8 +443,8 @@ <h3>Identifier Shadowing</h3>
 
 For some circumstances, when identifiers are shadowed by variables
 declared with short variable declarations,
-some new gophers may get confused on whether or not a variable
-in a short variable declaration is new declared.

```

Comment:
Here, "new" may be not perfect, but "now" is not better. How about change it to
```
... confused about whether a variable in a short variable declaration is redeclared or newly declared.
```

`redeclare` is mentioned in a piece of comment in the "Constants and Variables" article.